packages/browserify/test/globalRef.js

```diff
@@ -115,8 +21,76 @@ test('globalRef - has only the expected global circular refs', async (t) => {
       'global',
       'globalThis'
     ],
+    testType: 'truthy'
+  })
+  const testResult = await runScenario({ scenario })
+  scenario.checkResult(t, Array.isArray(testResult), scenario)
+  scenario.expectedResult.sort()
+  scenario.testType = 'deepEqual'
+  scenario.checkResult(t, testResult.sort(), scenario)
```

Comment:
use runAndTestScenario but override checkResult to sort the result. expected can be sorted at instantiation